src/bootman/bootman.c

```diff
@@ -63,6 +65,61 @@ BootManager *boot_manager_new()
         return r;
 }
 
+/**
+ * Set the command line to append into cmdline options on a global basis
+ */
+static void boot_manager_load_cmdline(BootManager *self)
+{
+        autofree(FILE) *f = NULL;
+        autofree(char) *cmdline = NULL;
+        size_t sn;
+        ssize_t r = 0;
+        char *buf = NULL;
+
+        if (self->cmdline) {
+                free(self->cmdline);
+                self->cmdline = NULL;
+        }
+
+        if (asprintf(&cmdline, "%s/etc/kernel/cmdline", self->sysconfig->prefix) < 0) {
+                DECLARE_OOM();
+                abort();
+        }
+
+        if (!nc_file_exists(cmdline)) {
+                return;
+        }
+
+        if (!(f = fopen(cmdline, "r"))) {
+                LOG_ERROR("Unable to open %s: %s", cmdline, strerror(errno));
+                return;
+        }
+
+        /* Keep cmdline in a single "line" with no spaces */
+        while ((r = getline(&buf, &sn, f)) > 0) {
+                char *tmp = NULL;
+                /* Strip newlines */
+                if (r > 1 && buf[r - 1] == '\n') {
```

Comment:
Ah yeah.  OK so we'll add a dedicated function for this as it seems this is our new "format" for config files :P